src/docker/install.ts

```diff
@@ -127,12 +130,28 @@ export class Install {
         const cli = await HubRepository.build('dockereng/cli-bin');
         extractFolder = await cli.extractImage(tag);
 
+        const moby = await HubRepository.build('moby/moby-bin');
         if (['win32', 'linux'].includes(platform)) {
           core.info(`Downloading dockerd from moby/moby-bin:${tag}`);
-          const moby = await HubRepository.build('moby/moby-bin');
           await moby.extractImage(tag, extractFolder);
         } else if (platform == 'darwin') {
-          // On macOS, the docker daemon binary will be downloaded inside the lima VM
+          // On macOS, the docker daemon binary will be downloaded inside the lima VM.
+          // However, we will get the exact git revision from the image config
+          // to get the matching systemd unit files.
+          core.info(`Getting git revision from moby/moby-bin:${tag}`);
+
+          // There's no macOS image for moby/moby-bin - a linux daemon is run inside lima.
+          const manifest = await moby.getPlatformManifest(tag, 'linux');
+
+          const config = await moby.getJSONBlob<Image>(manifest.config.digest);
+          core.info(`Config ${JSON.stringify(config.config)}`);
```

Comment:
Either remove this one or put it in debug mode: `core.debug`

Reply:
Ah good point, done